controllers/contacts.js

```diff
@@ -42,7 +41,7 @@ const updateContact = async (contactId, body) => {
   if (findIndex === -1) {
     return null;
   };
-  
+
   list[findIndex] = { ...list[findIndex], ...body };
 
   await fs.writeFilte("./contacts.json", JSON.stringify(list));
```

Comment:
literówka w metodzie `writeFilte` -> `writeFile`, w innych funkcjach `addContact` , `removeContact` występuje to samo